verified_model/Site/North/corsika_observation_level.json

```diff
@@ -0,0 +1,75 @@
+{
+    "value": 2156.0,
+    "units": "m",
+    "type": "numpy.float64",
+    "file": false,
+    "metadata": {
+        "CTA": {
+            "REFERENCE": {
+                "VERSION": "1.0.0"
+            },
+            "CONTACT": {
+                "ORGANIZATION": "CTAO",
+                "NAME": "CTAO Simulation Team",
+                "EMAIL": null
+            },
+            "PRODUCT": {
+                "DESCRIPTION": "Corsika observation level as used in prod6",
+                "CREATION_TIME": "2024-01-11T08:20:01",
+                "ID": null,
+                "DATA": {
+                    "CATEGORY": "SIM",
+                    "ASSOCIATION": "Subarray",
+                    "LEVEL": "R1",
+                    "TYPE": "Service",
+                    "MODEL": {
+                        "NAME": "corsika_observation_level",
+                        "VERSION": "0.1.0",
+                        "URL": "https://raw.githubusercontent.com/gammasim/simulation_model/main/schema/corsika_observation_level.schema.yml"
+                    }
+                },
+                "FORMAT": "json",
+                "VALID": {
+                    "START": null,
+                    "END": null
+                },
+                "FILENAME": "corsika_observation_level.json"
```

Comment:
What does this filename actually stand for? Is it the file that was used in the input of the data? Do we care about this filename?

Reply:
I am trying to understand where `FILENAME` comes from :-)

- it is not in the top-level metadata definition of PRODUCT:DATA
- I think we introduced it as it is useful for cases where we have metadata and data in different files

As this is not required by the top-level metadata, I agree that we can remove it in this context with data files with metadata included. I've checked the [simtools/schemas/metadata.metaschema.yml](https://github.com/gammasim/simtools/blob/main/simtools/schemas/metadata.metaschema.yml), and it is not in the list of required keys.

Removed `FILENAME` everywhere.

Reply:
OK!